.env.example

```diff
@@ -0,0 +1,2 @@
+SECRET_KEY=
+API_KEY=
```

Comment:
Environment variables are better than js objects

Reply:
@alphaolomi  okay,let me clean this up and add environment variables